spec/tasks/gen/model_spec.cr

```diff
@@ -0,0 +1,36 @@
+require "../../spec_helper"
+
+include CleanupHelper
+
+describe Gen::Model do
+  it "generates a model" do
+    with_cleanup do
+      io = IO::Memory.new
+      model_name = "User"
```

Comment:
Could you change the model name to `ContactInfo`? That way we can test that it creates the correct folder name `contact_info`.

Reply:
Great idea 👍